sim/pokemon.ts

```diff
@@ -214,6 +214,11 @@ export class Pokemon {
 		this.side = side;
 		this.battle = side.battle;
 
+		/**
+		 * An object for storing untyped data, for mods to use.
+		 */
```

Comment:
This should be on the `m: {...}` property description.

Reply:
Replacing the previous description I assume...